src/components/ExceededCommentLength.tsx

```diff
@@ -1,23 +1,22 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import useLocalize from '@hooks/useLocalize';
 import useThemeStyles from '@hooks/useThemeStyles';
 import CONST from '@src/CONST';
 import Text from './Text';
 
-const propTypes = {
-    shouldShowError: PropTypes.bool.isRequired,
+type ExceededCommentLengthProps = {
+    /** Show error when comment length is exceeded */
+    shouldShowError: boolean;
 };
 
-const defaultProps = {};
-
-function ExceededCommentLength(props) {
+function ExceededCommentLength({shouldShowError}: ExceededCommentLengthProps) {
```

Comment:
I know this is a migration issue, but If feel like a component called `ExceededCommentLength` shouldn't check on a shouldShowError boolean to display a message about a comment exceeding the max length.

Instead, we should check `shouldShowError` outside of the component, and only render it if it's true
```suggestion
function ExceededCommentLength() {
```